models/demos/t3000/falcon40b/demo/demo.py

```diff
@@ -23,12 +23,13 @@
 from models.utility_functions import (
     disable_compilation_reports,
     enable_persistent_kernel_cache,
-    profiler,
-    torch2tt_tensor,
-    tt_tensors_to_torch_tensors,
+    # profiler,
     nearest_32,
 )
 
+from models.perf.benchmarking_utils import BenchmarkProfiler
+from models.demos.utils.llm_demo_utils import create_benchmark_data, verify_perf, check_tokens_match
```

Comment:
It seems like check_tokens_match is not currently used, could you use it as well to generate `measurements["token_verification"]` similar to how it's done in Falcon7b?